tests/EvmAcceptanceTests/helpers/Parallelizer.ts

```diff
@@ -30,7 +30,11 @@ export class Parallelizer {
 
     web3.eth.getAccounts();
     const Contract = await hh_ethers.getContractFactory(contractName);
-    return Contract.connect(signer).deploy(...args);
+    const deployedContract = await Contract.connect(signer).deploy(...args);
+    if (process.env.ETHERNAL_PASSWORD) {
```

Comment:
Can you please change this `if` with a more high-level condition? Something like `hre.isEthernalEnabled` or something? You can extend `hre` using the `AddConfigHelpersToHre.ts` file